<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Circle_tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Circle_tab">
     <attribute name="title">
      <string>Circle_Tab</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>95</width>
        <height>158</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="Start_pushButton_2">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="step_pushButton">
         <property name="text">
          <string>step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Stop_pushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>半徑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Diameter_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>演算法</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Algorithm_comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Time_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>63</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>581</width>
        <height>541</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="PaintFrame" name="Paint_frame">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Diameter_horizontalSlider">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TickPosition::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Wave_tab">
     <attribute name="title">
      <string>Wave_Tab</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>95</width>
        <height>179</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="Wave_Start_pushButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Wave_step_pushButton">
         <property name="text">
          <string>step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Wave_Stop_pushButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>頻率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Frequency_comboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>協波</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Harmonic_comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="PaintFrame" name="Circle_frame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="PaintFrame" name="Sin_frame">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>225</width>
        <height>450</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="PaintFrame" name="cos_frame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>460</y>
        <width>450</width>
        <height>225</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="towDtheD_radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>94</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>2D/3D</string>
      </property>
     </widget>
     <widget class="QSlider" name="draw_Speed_Slider">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>710</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="draw_Speed_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>690</y>
        <width>53</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintFrame</class>
   <extends>QFrame</extends>
   <header location="global">PaintFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
